packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldEditor.ts

```diff
@@ -22,32 +22,46 @@ import type {
 import { splitMark } from "./utils.js";
 
 export interface SequenceFieldEditor extends FieldEditor<Changeset> {
-	insert(index: number, count: number, firstId: ChangesetLocalId): Changeset;
-	remove(index: number, count: number, id: ChangesetLocalId): Changeset;
-	revive(index: number, count: number, detachEvent: CellId, isIntention?: true): Changeset;
+	insert(index: number, count: number, firstId: CellId, revision: RevisionTag): Changeset;
+	remove(index: number, count: number, id: ChangesetLocalId, revision: RevisionTag): Changeset;
+	revive(
+		index: number,
+		count: number,
+		detachEvent: CellId,
+		revision: RevisionTag,
+		isIntention?: true,
+	): Changeset;
 
 	/**
 	 *
 	 * @param sourceIndex - The index of the first node move
 	 * @param count - The number of nodes to move
 	 * @param destIndex - The index the nodes should be moved to, interpreted before detaching the moved nodes
 	 * @param detachCellId - The local ID to assign to the first cell being emptied by the move
-	 * @param attachCellId - The local ID to assign to the first cell being filled by the move
+	 * @param attachCellId - The ID to assign to the first cell being filled by the move
+	 * @param revision - The revision to assign to the move marks
 	 */
 	move(
 		sourceIndex: number,
 		count: number,
 		destIndex: number,
 		detachCellId: ChangesetLocalId,
-		attachCellId: ChangesetLocalId,
+		attachCellId: CellId,
```

Comment:
It would be nice to keep `attachCellId` as a `ChangesetLocalId` (like `detachCellId`) because we don't support `attachCellId.revision` being different from `revision`.